.env

```
@@ -92,10 +92,10 @@ DEVTOOLSET_VERSION=
 # Used through docker-compose.yml and serves as the default version for the
 # ci/scripts/install_vcpkg.sh script. Prefer to use short SHAs to keep the
 # docker tags more readable.
-VCPKG="501db0f17ef6df184fcdbfbe0f87cde2313b6ab1"    # 2023.04.15 Release
+VCPKG="a42af01b72c28a8e1d7b48107b33e4f286a55ef6"    # 2023.11.20 Release
 
 # This must be updated when we update
-# ci/docker/python-wheel-windows-vs2017.dockerfile.
+# ci/docker/python-wheel-windows-vs2019.dockerfile.
 # This is a workaround for our CI problem that "archery docker build" doesn't
 # use pulled built images in dev/tasks/python-wheels/github.windows.yml.
 PYTHON_WHEEL_WINDOWS_IMAGE_REVISION=2023-08-02
```

We don't have to update this as we are already updating the image from `vs2017` to `vs2019` but maybe is worth to have an updated date for the `IMAGE_REVISION`. I would suggest the following change:
```suggestion
PYTHON_WHEEL_WINDOWS_IMAGE_REVISION=2024-02-05
```

---

Sounds sensible. (I have not tested locally)